bin/hdfcoinc/pycbc_page_segtable

```diff
@@ -108,8 +108,11 @@ for segment_name in opts.segment_names:
 
     # put comment in caption
     caption += " " + segment_name
-    if comment_dict[key] != None:
-        caption += " ("+comment_dict[key]+")"
```

Comment:
Okay, I'll mention again here that the SegFile class currently does not know about the comment attribute, and if this is going to be used it would be better added there than by using a separate function to read the XML file again to get this. Especially, as this class doesn't know about the caption attribute, it cannot *write* XML files with this, so I don't know how any of the segment files would get this!

Anyway, let me rebase this against master, and then get this ready to be pushed. Once pushed,  please open a PR with these changes against master. Please also consider that the SegFile class does know about which ifos are active in the segment file, so you shouldn't be hardcoding "H1,L1" as is currently done, especially with "V1" coming back online soon.